phpstan.neon

```diff
@@ -3,14 +3,12 @@ includes:
 
 parameters:
   level: 0
-  autoload_files:
-    - %rootDir%/../../../autoload.php
-#  autoload_directories:
-#    - %rootDir%/../../../vendor/bin/.phpunit/phpunit-5.7/src
+  paths:
+    - src
   excludes_analyse:
-    - %rootDir%/../../../src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/
-    - %rootDir%/../../../src/*/Tests/*
-    - %rootDir%/../../../src/Common/WebTestCase.php
+    - src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/
```

Comment:
Removing `rootDir` because it's no longer useful: https://phpstan.org/config-reference#expanding-paths

> These are no longer very useful as all relative paths in config parameters are resolved based on the directory of the config file is in, resulting in a short and concise notation.